views/error.hbs

```diff
@@ -26,18 +33,24 @@
       <p>Error reference: {{{errorReference}}}</p>
       {{/if}}
 
-      <p class="error-message">
-        We track these errors automatically, but if the problem persists feel
-        free to
-        <a
-          href="https://support.atlassian.com/contact/#/?inquiry_category=technical_issues&is_cloud=true&product_key=third-party-product"
-        >contact us</a>. In the meantime, try adding an organization again in
-        Jira.
-      </p>
+      {{#if ctaUrl}}
+          <a class="aui-button aui-button-primary m-4" href="{{ctaUrl.action}}">
```

Comment:
When you update the middleware this should just be `href="{{ctaUrl}}"` and `{{ctaUrl.text}}` should just be `Retry`

Reply:
updated